packages/Interface/src/DTK_Fortran_API.f90

```diff
@@ -111,6 +112,7 @@ subroutine swigc_DTK_set_function(farg1, farg2, farg3, farg4) &
    type(C_PTR), value :: farg4
   end subroutine
  end interface
+#endif
```

Comment:
I don't think Breathe (Sphinx plugin to import Doxygen output) is responsible for that.
The XML output generated by Doxygen does have the functions prefixed by swig_ even though the Doxyfile (docs/build/breathe/doxygen/dtk.cfg) has both `ENABLE_PREPROCESSING = YES` and `PREDEFINED=DOXYGEN_SHOULD_SKIP_THIS`.
```bash
$ grep -rc swig docs/build/breathe/doxygen/dtk/xml/index.xml
docs/build/breathe/doxygen/dtk/xml/index.xml:18
```